selftests/functional/test_replay_basic.py

```diff
@@ -57,6 +57,13 @@ def test_run_replay_latest(self):
         expected_rc = exit_codes.AVOCADO_ALL_OK
         self.run_and_check(cmd_line, expected_rc)
 
+    def test_run_replay_jobdir(self):
```

Comment:
Some dosctrings could help in this (well, everywhere in our selftests). It took me a while to see the difference.